force-app/main/default/classes/CellOrderTriggerHelper.cls

```diff
@@ -0,0 +1,35 @@
+public without sharing class CellOrderTriggerHelper {
+    public CellOrderTriggerHelper() {}
+    public void validateParcelDimensions(List<CellOrder__c> newOrders) {
+        for (CellOrder__c cellOrder : newOrders) {
+        // Retrieving related objects
+        Cell__c cell = [SELECT Width__c, Height__c, Depth__c FROM Cell__c WHERE Id = :cellOrder.Cell__c];
```

Comment:
Cell Order could not have Cell value when we are going to start delivery. Let's skip records that have Cell field blank.